nrich-jackson-spring-boot-starter/README.md

```diff
@@ -1,3 +1,84 @@
 # nrich-jackson-spring-boot-starter
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/net.croz.nrich/nrich-jackson-spring-boot-starter/badge.svg?color=blue)](https://maven-badges.herokuapp.com/maven-central/net.croz.nrich/nrich-jackson-spring-boot-starter)
+
+## Overview
+
+Spring Boot starter for nrich-jackson. The purpose of `nrich-form-jackson` is to provide additional Jackson's serializers and deserializers.
```

Comment:
I think it should be "Spring Boot starter for `nrich-jackson` module". We agreed about this earlier. You probably missed it here. 

Additionally, I think the module name (`nrich-form-jackson` instead of `nrich-jackson`) is not correct in a few places.